project/Build.scala

```diff
@@ -13,14 +13,14 @@ import com.typesafe.sbt.SbtGit.useJGit
 
 object BuildSettings {
   val buildOrganization = "ch.unibas.cs.gravis"
-  val buildScalaVersion = "2.11.7"
+  val buildScalaVersion = "2.12.1"
```

Comment:
I think we should list both versions (2.11.7 and 2.12.1) as ```crossScalaVersions``` below and use 2.11.7 as the default version for the compilation. In this way we are not likely to use language features that are not supported by scala 2.11.